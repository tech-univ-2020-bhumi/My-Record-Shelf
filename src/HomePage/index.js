/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import propTypes from 'prop-types';
//  import { FaHeart } from "react-icons/fa";

function HomePage(props) {
  const { testId } = props;
  return (
    <div data-tetid={testId}>
      <div>My Record Shelf</div>
      <div>
        :((
        <br />
        seems a bit empty in here...
      </div>
    </div>

  );
}

HomePage.propTypes = {
  testId: propTypes.func.isRequired,
};

export default HomePage;
