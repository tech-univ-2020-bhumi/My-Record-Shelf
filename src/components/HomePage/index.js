/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import propTypes from 'prop-types';
import { Link } from 'react-router-dom';
import SyncButton from '../SyncButton';
//  import { FaHeart } from "react-icons/fa";

function HomePage(props) {
  const { testId } = props;
  return (
    <div data-tetid={testId}>
      <div>My Record Shelf</div>
      <div>
        :((
        <br />
        seems a bit empty in here...
        {/* <Link to="/genres"> */}
        <div><SyncButton /></div>
        {/* </Link> */}
      </div>
    </div>

  );
}

HomePage.propTypes = {
  testId: propTypes.func.isRequired,
};

export default HomePage;
