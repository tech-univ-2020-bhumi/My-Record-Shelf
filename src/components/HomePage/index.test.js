/* eslint-disable react/jsx-filename-extension */
/* eslint-disable no-undef */
import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import HomePage from './index';

describe('The homePage component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<HomePage testId="123" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
