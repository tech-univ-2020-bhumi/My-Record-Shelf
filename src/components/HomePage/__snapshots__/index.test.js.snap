// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The HomePage component should render correctly 1`] = `
<DocumentFragment>
  <div
    data-tetid="123"
  >
    <div>
      My Record Shelf
    </div>
    <div>
      :((
      <br />
      seems a bit empty in here...
      <div>
        <button
          type="button"
        >
          sync
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
